dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploaderTest.kt

```diff
@@ -275,6 +279,105 @@ internal class OkHttpUploaderTest {
             )
     }
 
+    @Test
+    fun `M throw a MaxSizeExceededException W upload() { mappingFile size exceeded 50 MB}`(
+        forge: Forge
+    ) {
+        // GIVEN
+        val fakeTooLargeMappingFile = spy(File(tempDir, forge.anAlphabeticalString()))
+        doReturn(1024L * 1024L * 50 + forge.anInt(min = 0))
+            .whenever(fakeTooLargeMappingFile).length()
+
+        // THEN
+        assertThrows<MaxSizeExceededException>(
+            {
+                testedUploader.upload(
+                    mockWebServer.url("/").toString(),
+                    fakeTooLargeMappingFile,
+                    fakeRepositoryFile,
+                    fakeIdentifier,
+                    fakeRepositoryInfo
+                )
+            }
+        )
+    }
+
+    @Test
+    fun `M throw a MaxSizeExceededException W upload() { mappingFile size between 50 and 51 MB}`(
```

Comment:
Technically a file size between 50 and 51MB is always greater than or equal to 50MB. I don't really see the point of having this additional test. I guess we can have three tests: size<50Mb, size==50Mb and size>50Mb.

Reply:
Oh...sorry I forgot to push a new commit where I cleaned up these tests. I will change though the logic following your suggestion and as it is much more simpler and will addd these 3 tests.